package.json

```diff
@@ -17,7 +17,6 @@
     },
     "./interactionsTests": "./dist/esm/src/tests/interactionsTests.js",
     "./testIds": "./dist/esm/src/tests/testIds.js",
-    "./storybookComponents": "./dist/storybook/index.js",
```

Comment:
We probably shouldn't remove this endpoint - it's a breaking change
I think we should consider doing it in the next major version